client/src/components/ElementEditor/Element.js

```diff
@@ -53,6 +73,67 @@ class Element extends Component {
         captureDraggingState: true,
       });
     }
+    // Check if formSchema state has already been loaded before opening a block
+    // This can happen if there was a validation error on a block after performing a Page save
+    if (this.props.formStateExists) {
+      this.setState({
+        formHasRendered: true
+      });
+    }
+  }
+
+  componentDidUpdate() {
+    if (this.state.justSavedElement) {
+      this.setState({
+        justSavedElement: false,
+      });
+      if (!this.state.doPublishElement) {
+        this.showSavedElementToast();
+      }
+      // This will trigger a graphql readOneElementalArea request that will cause this
+      // element to re-render including any updated title and versioned badge
+      window.ss.apolloClient.queryManager.reFetchObservableQueries();
+    }
+    if (this.state.doSetFormChanged) {
+      this.setState({
+        doSetFormChanged: false,
+      });
+      this.props.setFormChanged();
+    }
+  }
+
+  showSavedElementToast() {
+    const title = this.state.newTitle || this.getNoTitle();
+    const message = i18n.inject(
+      i18n._t('ElementSaveAction.SUCCESS_NOTIFICATION', 'Saved \'{title}\' successfully'),
+      { title }
+    );
+    this.props.actions.toasts.success(message);
+  }
```

Comment:
We now do a remote-submit of redux-form which doesn't return a promise. Because of this we can't monitor if for a 400+ http status code which is what the old catch() block used to do when doing a fetch via lib/Backend

Reply:
I challenge the argument that it would be a separate card - this PR is breaking that functionality, so if we want to keep it we should do it as part of this card.

The one question I have is: If we **could** detect that failed request, would it include failures as a result of validation errors?
If so, we need to fix it so we get toasts for validation errors.
If not, I'm happy for that to be left as is.

Reply:
>  If we could detect that failed request, would it include failures as a result of validation errors?

No, the old toast was for 400-500 http status code failures such as failed permission check (403), not validation errors which come back in a 200. That's possible incorrect (maybe it should be 400), though linkfield is also returning 200 on validation error so should stick with that.